packages/mesh/src/Mesh.ts

```diff
@@ -399,7 +427,7 @@ export class Mesh extends Container
         }
         else
         {
-            this.uvs = geomUvs.data;
+            this.uvs = geomUvs.data as Float32Array;
```

Comment:
We can use `ITypedArray` in such places in future, but its fine to convert it if we know certain type.